.github/workflows/nightly.yml

```diff
@@ -141,9 +141,29 @@ jobs:
 
   nightly_changelog:
     runs-on: ubuntu-latest
+    env:
+      HIGHCHARTS_VISUAL_TESTS_BUCKET: ${{secrets.VISUAL_TESTS_S3_BUCKET}}
+      AWS_ACCESS_KEY_ID: ${{secrets.VISUAL_TESTS_AWS_ACCESS_KEY_ID}}
+      AWS_SECRET_ACCESS_KEY: ${{secrets.VISUAL_TESTS_AWS_SECRET_ACCESS_KEY}}
+      AWS_REGION: ${{secrets.VISUAL_TESTS_AWS_REGION}}
     steps:
       - name: Checkout repo
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # get the full history
+
+      - run: npm i
+
+      - name: Build tree.json
+        run: |
+          npx gulp scripts
+          npx gulp jsdoc-options
+
+      - name: Generate and upload nightly changelog
+        run: |
+          node changelog/generate --since $(git describe --abbrev=0 --match 'v*') --review
```

Comment:
Do you need the `--since` argument? It defaults to the latest release. I see that the description on line 10 is wrong, it says it defaults to the latest commit, which makes no sense.

Reply:
Looks like no (and since octokit also prefers annotated tags we don't have to filter out Dashboards releases)